pyproject.toml

```diff
@@ -642,5 +646,5 @@ exclude_lines = [
 directory = "coverage_html_report"
 
 [tool.codespell]
-ignore-words-list = "blocs, coo, hist, nd, sav, ser, recuse, nin, timere"
+ignore-words-list = "blocs, coo, hist, nd, sav, ser, recuse, nin, timere, afile, foor, aas, mis"
```

Comment:
Where are these new ignore words found?

Github code search doesn't seem to find them.

Reply:
These are found in the tests, either partially or fully in string variables

Reply:
Hm, might be good to add a comment or rename some of the variables.

Reply:
Sure thing. I'll just rename some variables